packages/code-studio/src/index.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { Suspense } from 'react';
 import ReactDOM from 'react-dom';
 import '@deephaven/components/scss/BaseStyleSheet.scss';
-import { LoadingOverlay } from '@deephaven/components';
+import { initializeTheme, LoadingOverlay } from '@deephaven/components';
 import { ApiBootstrap } from '@deephaven/jsapi-bootstrap';
 import logInit from './log/LogInit';
 
 logInit();
 
+initializeTheme();
```

Comment:
We might want to inline and call this function in a script tag in the actual html <head>, that way there's no flash of the wrong theme before react loads/initializes.

Reply:
I'm fine with that. Are you actually seeing a flash?